invalidate-cloudfront.js

```diff
@@ -21,21 +22,21 @@ if (shouldDoInvalidation()) {
         })
         .send((error, success) => {
             if (error) {
-                console.log(colors.red('ERROR WHILE INVALIDATING RESSOURCES ON CLOUDFRONT'));
-                console.log(colors.red('*************'));
-                console.log(colors.red(error.message));
-                console.log(colors.red('*************'));
+                console.log(logInRed('ERROR WHILE INVALIDATING RESSOURCES ON CLOUDFRONT'));
```

Comment:
```suggestion
                logInRed('ERROR WHILE INVALIDATING RESSOURCES ON CLOUDFRONT');
```
Same thing for all other lines. You already have a console.log in there